hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/scale/AbstractSTestS3AHugeFiles.java

```diff
@@ -482,7 +484,7 @@ public void test_050_readHugeFile() throws Throwable {
    */
   @Test
   public void test_090_verifyRenameSourceEncryption() throws IOException {
-    if(isEncrypted(getFileSystem())) {
+    if (isEncrypted(getFileSystem())) {
```

Comment:
I would avoid adding diffs like this if you haven't already touched code in the adjacent lines

Reply:
I think the IDE formatted it and it looks good to me. I will avoid it next time.